src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -59,9 +63,79 @@
     ProjectUptimeSubscriptionMode,
     UptimeStatus,
 )
+from sentry.users.models.user import User
 from sentry.utils.cursors import Cursor, StringCursor
 
 
+def create_metric_alert(
```

Comment:
Need to change this from class method to standalone helper method so we can re-use in the project signal